doc/dev/crimson/crimson.rst

```diff
@@ -146,6 +124,106 @@ Running Crimson
 
 As you might expect, Crimson does not yet have as extensive a feature set as does ``ceph-osd``.
 
+vstart.sh
+=========
+
+The following options can be used with ``vstart.sh``.
+
+``--crimson``
+    Start ``crimson-osd`` instead of ``ceph-osd``.
+
+``--nodaemon``
+    Do not daemonize the service.
+
+``--redirect-output``
+    Redirect the ``stdout`` and ``stderr`` to ``out/$type.$num.stdout``.
```

Comment:
s/the//

Reply:
I didn't change this part, could you be a bit more specific please so I can be sure to update the relevant part correctly - I see a double // i. your comment